checker/src/main/java/org/checkerframework/checker/calledmethods/CalledMethodsTransfer.java

```diff
@@ -134,20 +129,19 @@ public TransferResult<AccumulationValue, AccumulationStore> visitMethodInvocatio
               .adjustMethodNameUsingValueChecker(methodName, node.getTree());
       accumulate(receiver, superResult, methodName);
     }
-    TransferResult<AccumulationValue, AccumulationStore> finalResult =
-        new ConditionalTransferResult<>(
-            superResult.getResultValue(),
-            superResult.getThenStore(),
-            superResult.getElseStore(),
-            exceptionalStores);
-    exceptionalStores = null;
-    return finalResult;
+    return new ConditionalTransferResult<>(
+        superResult.getResultValue(),
+        superResult.getThenStore(),
+        superResult.getElseStore(),
+        exceptionalStores);
   }
 
   @Override
   public void accumulate(
       Node node, TransferResult<AccumulationValue, AccumulationStore> result, String... values) {
     super.accumulate(node, result, values);
+
+    @Nullable Map<TypeMirror, AccumulationStore> exceptionalStores = result.getExceptionalStores();
```

Comment:
We shouldn't need an explicit `@Nullable` annotation here?  It should be inferred.

Reply:
You are right! I'll fix this up soon.